src/createStore.js

```diff
@@ -125,6 +127,11 @@ export default function createStore(reducer, initialState) {
     dispatch({ type: ActionTypes.INIT });
   }
 
+  /**
```

Comment:
Hmm good question. No it's not meant to be JSDoc, just a comment to explain why we are dispatching. Should I use `//` line comments or should I just remove it? What do you suggest?